packages/vscode-extension/src/common/Project.ts

```diff
@@ -15,6 +15,10 @@ export type DeviceSettings = {
   locale: Locale;
   replaysEnabled: boolean;
   showTouches: boolean;
+  camera: {
```

Comment:
Since we only use this on Android, I think this should be optional (or otherwise typed so that we can leave this unset on iOS, instead of setting it to some arbitrary value that we never use, like we do now.)